### ISSUE TITLE
No warning for too small hatchery

### ISSUE THREAD

**Issue by Loobinex** [OpenDungeons/OpenDungeons#1238]
I played a game in version git20160725 and suddenly I had several creatures angry and I did not know why. After investigating a bit I found out that my creatures were hungry and I needed a larger hatchery.

It would have been very nice to receive a warning for this.

In case I should have, I attached the log file and replay of this game. I played just a single game in git20160725 so these are also for the other issues raised today.

[opendungeons.log.zip](https://github.com/OpenDungeons/OpenDungeons/files/385038/opendungeons.log.zip)
[replay_20160726_235304.zip](https://github.com/OpenDungeons/OpenDungeons/files/385035/replay_20160726_235304.zip)

**Comment by hwoarangmy** [comment]
Concerning hunger, there is a mood icon so it should be easy to see that creatures are hungry.

Concerning room size, I would go for something more generic like displaying a message when a room is at full capacity for too long.

**Comment by Loobinex** [comment]
Yes, but I was busy digging tunnels toward the enemy when the creatures at home turned angry. I had to go back to my dungeon and search around for angry creatures. #1239 would help, but I'd like to be warned that I'm out of chickens before my creatures become unhappy.

A simple (voice) message "You need a bigger hatchery" would suffice.

**Comment by akien-mga** [comment]
Just a side note, we should make sure that we have a consistent design for visual and audio feedback that we give to the players. There are tons of things that need to be conveyed, and we already have quite a few different mechanisms:
- Overlay icons
- Text messages
- Audio messages

We need to ensure that we're using for each kind of feedback the proper mechanism and that it conveys the information efficiently while keeping the game funny.
